### DISCUSSION TITLE
S3 환경 구성

### DISCUSSION THREAD

**Discussion by duri-wip** [DEProjTeam07/coc-pipeline#20] [General]
__버킷 1__ 
이름 : deprojteam07-labeledrawdata
속성 :
- ACL 비활성화
- 모든 퍼블릭 액세스 차단 ( 버킷 정책 수정을 통해 필요한 접근만 허용)
- 버킷 버전관리 비활성화
- S3관리형키(SSE-S3)를 통한 암호화
- 버킷 키 활성화 -> 버킷에 저장되는 모든 객체 암호화해서 저장해줌

__버킷 2__
이름 : deprojteam07-datalake
속성 :
- ACL 비활성화
- 모든 퍼블릭 액세스 차단 ( 버킷 정책 수정을 통해 필요한 접근만 허용)
- 버킷 버전관리 활성화
- S3관리형키(SSE-S3)를 통한 암호화
- 버킷 키 활성화 -> 버킷에 저장되는 모든 객체 암호화해서 저장해줌

**Comment by duri-wip** [comment]
버킷 정책 수정 시 참고할 자료 : [가이드](https://docs.aws.amazon.com/ko_kr/AmazonS3/latest/userguide/example-bucket-policies.html?icmpid=docs_amazons3_console)

**Comment by dnlpys** [comment]
인스턴스1,2,4번에 코치님께 요청해서 AmazonS3FullAccess권한 부여해놨습니다.
ec2인스턴스에 iam권한 부여하면, 해당 ec2에서 인증 없이 s3에 접근가능하고 합니다.

**Comment by Kdreamtomaster** [comment]
deprojteam07-airflowdagtest 
airflow dag test 용 버킷 생성하였습니다.  테스트 끝나면 삭제하도록 하겠습니다.
